pandas/plotting/_matplotlib/boxplot.py

```diff
@@ -34,7 +35,10 @@
 
 
 class BoxPlot(LinePlot):
-    _kind = "box"
+    @property
+    def _kind(self) -> Literal["box"]:
+        return "box"
```

Comment:
this change causes the `TypeError: cannot pickle 'property' object`. Properties are usually pickable, not sure why this one is causing issues.